bin/tigmint-make

```diff
@@ -149,11 +149,32 @@ tigmint: \
 	$(draft).$(reads).as$(as).nm$(nm).molecule.size$(minsize).trim$(trim).window$(window).span$(span).breaktigs.fa \
 	$(draft).tigmint.fa
 
+# Index the linked reads 
+tigmint-index: $(draft).fa.bwt
+
+# Align the linked reads (using bwa alignment) 
+tigmint-align: $(draft).$(reads).sortbx.bam
+
+# Perform the tigmint-molecule step for linked reads 
+tigmint-molecule: $(draft).$(reads).as$(as).nm$(nm).molecule.size$(minsize).bed
+
+# Perform the final tigmint-cut step for linked reads
+tigmint-cut: $(draft).$(reads).as$(as).nm$(nm).molecule.size$(minsize).trim$(trim).window$(window).span$(span).breaktigs.fa
+
 # Correct misassemblies for long reads using Tigmint
 tigmint-long: \
 	$(draft).$(reads).cut$(cut).molecule.size$(minsize).trim$(trim).window$(window).span$(span).breaktigs.fa \
 	$(draft).cut$(cut).tigmint.fa
 
+# Estimate the tigmint parameters for long reads 
+tigmint-long-estimate: $(reads).tigmint-long.params.tsv
+
+# # Covert the long reads to linked reads, perform minimap alignment and the tigmint-molecule step
+tigmint-long-to-link: $(draft).$(reads).cut$(cut).molecule.size$(minsize).bed
```

Comment:
I'm finding in testing on my end that if I run this target `tigmint-long-to-link` twice, it will execute twice vs. recognizing that the step has already been completed. Could you look into troubleshooting? You should see if you run another rule twice in a row it should say "nothing to be done" the second time. The `-d` option is useful for debugging this